carbon-design-system/carbon-addons-iot-react
# fix(table): fix select all checkbox focus

Closes #3139

**Summary**
The focus border of the Table select all checkbox was cut off by the overflow of the parent being hidden. 

**Change List (commits, features, bugs, etc)**

- Add overflow: visible to the container of the checkbox

**Acceptance Test (how to verify the PR)**

1. go to https://deploy-preview-3166--carbon-addons-iot-react.netlify.app??path=/story/1-watson-iot-table-table--basic-dumb-table
2. click on the select all checkbox
3. make sure the blue focus border is completely visible

**Regression Test (how to make sure this PR doesn't break old functionality)**

Make sure the checkbox looks ok in all other table configurations, including but not limited to hasResize:true.



**Things to look for during review**
